_posts/2020-09-16-cockpit-228.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Cockpit 228
+author: kkoukiou
+date: 2020-09-16
+tags: cockpit linux
+slug: cockpit-228
+category: release
+summary: Cockpit
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly. Here are the release notes from Cockpit version 228.
+
+### Accounts: Allow setting weak passwords
+
+Cockpit now allows users to set up a weak password. The user gets notified that
+the password is weak, but clicking the submit button again will use the selected password anyway.
+This is similar to how weak passwords are handled in the Anaconda installer.
+
+![Accounts weak password](/images/accounts-weak-password.png){:.screenshot}
+
+### Changes to remote host logins
+
+Cockpit now never tries to log into remote hosts with your initial login password,
+and consequently does not promote password reuse across accounts anymore.
+Instead, Cockpit now helps with setting up SSH keys for users that want automatic
+and password-less login to remote hosts.
+
+![Remote host logins](/images/remote-hosts-logins.png){:.screenshot}
```

Comment:
The [last demo video](https://www.youtube.com/watch?v=4UBZWKk-T28) is reasonably close to the current experience, just a tad sloppy at the beginning. We could link to it saying that it's a demo, just perhaps not show it inline? It gives a much better demonstration what this is all about than a simple screenshot.

Reply:
I put back the screenshot, and still keep the demo video link.